metashare/api/sf_run_flow.py

```diff
@@ -304,3 +304,26 @@ def create_org_and_run_flow(
         flow.run(org_config)
 
     return scratch_org_config
+
+
+def delete_scratch_org(scratch_org):
+    # Update environment with some keys CumulusCI uses
+    # TODO: Should this be in a context manager so as not to leak across
+    # runs?
+    os.environ.update(
+        {
+            # Salesforce connected app
+            "SFDX_CLIENT_ID": SF_CLIENT_ID,
+            "SFDX_HUB_KEY": SF_CLIENT_KEY,
+        }
+    )
```

Comment:
We shouldn't need to set these environment variables at runtime.

Reply:
Do we need to set them up on L237, at the start of `create_org_and_run_flow`? I imagine not, and would love to remove them there too.

Reply:
Like I said in slack, I think we're bypassing the places in CumulusCI where they are used. But in order to be safe I would recommend setting these in the MetaShare environment outside of runtime (i.e. rename the existing SF_CLIENT_ID and SF_CLIENT_KEY env vars to SFDX_CLIENT_ID and SFDX_CLIENT_KEY)

Hmm, except CumulusCI doesn't know how to deal with the escaped newlines in the key.

Reply:
Which, unfortunately, are the only way Docker can inject them. It should work on Heroku, but it'll be a mess in local dev.

Reply:
We could update the environment to set SFDX_CLIENT_KEY based on the processed SF_CLIENT_KEY in the settings module at import time rather than doing it at runtime here. Let's wait until we discover that we have a real need for it though.